clang/include/clang/Basic/LangOptions.def

```diff
@@ -221,6 +221,7 @@ LANGOPT(CUDADeviceApproxTranscendentals, 1, 0, "using approximate transcendental
 LANGOPT(GPURelocatableDeviceCode, 1, 0, "generate relocatable device code")
 
 LANGOPT(SYCLIsDevice      , 1, 0, "Generate code for SYCL device")
+LANGOPT(SYCLIsHost          , 1, 0, "SYCL host compilation")
```

Comment:
Alignment is bit off here.

Reply:
Thanks for taking a look Erich and Alexey. 

@bader I've corrected the alignment in new patch